RELEASE-NOTES.md

```diff
@@ -1,5 +1,26 @@
 ### next
 
+### v0.3.0
```

Comment:
There is no lack of information about this?
https://github.com/Code-Poets/sheetstorm/pull/123

Reply:
You're right. Fixed